### ISSUE TITLE
app parameter results in black screen

### ISSUE THREAD

**Issue by guy032** [Niap/node-freerdp2#1]
Any idea why the app parameter for RemoteApp result in black screen? Desktop mode works as expected.
@Niap Contact me at guy032@gmail.com (Guy Mesika) please.
My company is willing to pay for your services.

**Comment by Niap** [comment]
Sorry...could you give me more information about "black screen"?

**Comment by guy032** [comment]
Yes. desktop experience works great but when I try to add this parameter /app:explorer for example or any other app like calc or notepad or full exe path I can see the login screen for a second and after the login I can only see blank (black) screen.
I no longer receive the bitmap events after the login screen when the black screen appears.
Apparently even if I try to test it with wfreerdp: https://cloudbase.it/freerdp-for-windows-nightly-builds/
it still doesn't work with their recent binaries but I can open task manager with keyboard inside the black screen.
the command used: wfreerdp.exe /v:{hostname} /u:{username} /p:{password} /app:calc
Only for Linux with xfreerdp2 I was capable to see this work but not from node-freerdp2.

@Niap
